docs/src/main/asciidoc/amazon-s3.adoc

```diff
@@ -39,9 +39,9 @@ The easiest way to start working with S3 is to run a local instance as a contain
 
 [source,shell,subs="verbatim,attributes"]
 ----
-docker run -it --publish 8008:4572 -e SERVICES=s3 -e START_WEB=0 localstack/localstack
+docker run -it --publish 8008:4566 -e SERVICES=s3 -e START_WEB=0 localstack/localstack
 ----
-This starts a S3 instance that is accessible on port `8008`.
+This starts a S3 instance that is accessible on port `8008`. **Note**: Versions of localstack newer than v0.11.5 require port `4566` instead of port `4572`. See this https://github.com/localstack/localstack/issues/2983[GitHub issue] for details on this change
```

Comment:
Minor nitpick: Can we have this on a separate line like so:

```
NOTE: Versions of localstack newer than v0.11.5....
```

This will make the rendering of the adoc as HTML much better.

Thanks!